src/hooks/useListenFocusTriggers/index.ts

```diff
@@ -20,9 +20,9 @@ export function useListenFocusTriggers({
     (e: FocusEvent) => {
       // if focus triggered by mouse down, call onFocusByMouse
       if (isElementMouseDown.current) {
-        onFocusByMouse(e);
+        onFocusByMouse && onFocusByMouse(e);
```

Comment:
```suggestion
        onFocusByMouse?.(e);
```

same to the line below